dockerfiles/docker-compose.yml

```diff
@@ -61,6 +63,7 @@ services:
     environment:
       - INIT=${INIT:-}
       - DOCKER_NO_RELOAD
+      - RTD_EXT_THEME_ENABLED
```

Comment:
Where this value come from? `inv docker.up` command with a flag?

Reply:
Yes, just from the task, but you could also specify in `.envrc` as a side effect